incubator/binding-risingwave/src/main/java/io/aklivity/zilla/runtime/binding/risingwave/internal/statement/RisingwaveCreateTableGenerator.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.binding.risingwave.internal.statement;
+
+import net.sf.jsqlparser.statement.Statement;
+import net.sf.jsqlparser.statement.create.table.CreateTable;
+
+public class RisingwaveCreateTableGenerator extends CommandGenerator
+{
+    private final String bootstrapServer;
+    private final String schemaRegistry;
+    private final long scanStartupMil;
+
+    private String table;
+    private String primaryKey;
+
+
+    public RisingwaveCreateTableGenerator(
+        String bootstrapServer,
+        String schemaRegistry,
+        long scanStartupMil)
+    {
+        this.bootstrapServer = bootstrapServer;
+        this.schemaRegistry = schemaRegistry;
+        this.scanStartupMil = scanStartupMil;
+    }
+
+    public String generate(
+        Statement statement)
+    {
+        CreateTable createTable = (CreateTable) statement;
+        this.table = createTable.getTable().getName();
+        this.primaryKey = getPrimaryKey(createTable);
+
+        return format();
```

Comment:
Why object level field state instead of passing parameters to `format()` method?